src/main.rs

```diff
@@ -1,79 +1,36 @@
 //! SUPER Android Analyzer
-
-#![forbid(deprecated, overflowing_literals, stable_features, trivial_casts, unconditional_recursion,
```

Comment:
Why have all these lints been removed?

Reply:
They are moved to `lib.rs` if I recall correctly. Thinking of it, they should actually stay in `main.rs` too. I'll put them back in a bit